modules/oadp-1-4-1-release-notes.adoc

```diff
@@ -0,0 +1,128 @@
+:_newdoc-version: 2.18.3
+:_template-generated: 2024-09-12
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-1-4-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="oadp-1-4-1-release-notes_{context}"]
+= OADP 1.4.1 release notes
+
+The {oadp-first} 1.4.1 release notes lists new features, resolved issues and bugs, and known issues.
+
+[id="new-features-1-4-1_{context}"]
+== New features
+
+.New DPA fields to update client qps and burst
+
+You now can change Velero Server Kubernetes API queries per second and burst values using new Data Protection Application (DPA) fields. The new DPA fields are `spec.configuration.velero.client-qps` and `spec.configuration.velero.client-burst`, which both default to 100. 
+link:https://issues.redhat.com/browse/OADP-4076[OADP-4076]
+
+
+[id="resolved-issues-1-4-1_{context}"]
+== Resolved issues
+
+.PodVolumeBackup CR now displays correct message
+
+Previously, the `PodVolumeBackup` CR had an incorrect message, `get a podvolumebackup with status "InProgress" during the server starting, mark it as "Failed"`.
+With this update, the message is now updated as follows:
+[source,terminal]
+----
+found a podvolumebackup with status "InProgress" during the server starting,
+mark it as "Failed".
+----
+link:https://issues.redhat.com/browse/OADP-4224[OADP-4224]
+
+.Overriding imagePullPolicy is now possible with DPA  
+
+Previously, OADP used to set the `imagePullPolicy` parameter to `Always` for all images.
+With this update, OADP now checks if each image contains `sha256` or `sha512` digest then it sets `imagePullPolicy` to `IfNotPresent`; otherwise sets it to `Always`. You can now override this policy by using the new `spec.containerImagePullPolicy` DPA field.
```

Comment:
```suggestion
With this update, {oadp-short} checks if each image contains `sha256` or `sha512` digest, then it sets `imagePullPolicy` to `IfNotPresent`; otherwise sets it to `Always`. You can now override this policy by using the new `spec.containerImagePullPolicy` DPA field.
```